src/snowflake/snowpark/_internal/utils.py

```diff
@@ -52,6 +54,13 @@
     f"^(({SNOWFLAKE_ID_PATTERN}\\.){{0,2}}|({SNOWFLAKE_ID_PATTERN}\\.\\.)){SNOWFLAKE_ID_PATTERN}$"
 )
 
+SNOWFLAKE_CASE_INSENSITIVE_QUOTED_ID_PATTERN_RE = re.compile(
```

Comment:
nit: just for consistency, let's name it `SNOWFLAKE_CASE_INSENSITIVE_QUOTED_ID_RE_PATTERN`? and also for another one